utils/_context/virtual_machines.py

```diff
@@ -653,7 +653,7 @@ def __init__(self, **kwargs) -> None:
             krunvm_config=None,
             os_type="linux",
             os_distro="rpm",
-            os_branch="redhat",
+            os_branch="redhat_9",
```

Comment:
If you change this you have to be careful with this: [link](https://github.com/DataDog/system-tests/blob/9baa358bdbf1e43d47fb370cdfd4978e47b85193/tests/auto_inject/test_blocklist_auto_inject.py#L223)

and with this: [link](https://github.com/DataDog/system-tests/blob/9baa358bdbf1e43d47fb370cdfd4978e47b85193/utils/build/virtual_machine/provisions/auto-inject/docker/auto-inject_prepare_docker.yml#L93)